x/marker/types/key.go

```diff
@@ -32,6 +32,9 @@ const (
 var (
 	// MarkerStoreKeyPrefix prefix for marker-address reference (improves iterator performance over auth accounts)
 	MarkerStoreKeyPrefix = []byte{0x02}
+
+	// DenySendKeyPrefix prefix for adding addresses that are denied send functionality on restricted markers
+	DenySendKeyPrefix = []byte{0x02}
```

Comment:
Need to have different values here (both are `0x02`).

Reply:
😬 Thought I changed that.  Good catch, that could have been an issue.

Reply:
curious why a test didn't fail with this ... definitely a good catch.

Reply:
Should not have written the test like 
```	
assert.Equal(t, DenySendKeyPrefix[0], denyKey[0], "should have correct prefix for deny send key")
` ``

Changed it to (which I was about to do in the first place...ugh): 
```
assert.Equal(t, uint8(3), denyKey[0], "should have correct prefix for deny send key")
```

Reply:
I thought this test was fine. Either way works though.

Since the key formats are different, it wouldn't have been overwriting existing data. However, iterating over all markers would have gone wrong when it hit a deny list entry and failed to marshal it.